wiki/Gameplay/Input_device/Dance_pad/en.md

```diff
@@ -0,0 +1,6 @@
+# Dance Pad
+
+A **Dance Pad** is a type of rhythm game controller that is placed on the floor and the user steps on the pad to activate the keys. 
+The Wii Dance Mat requires a GameCube controller adaptor to function on PC. Once you have connected the dance pad to your PC, you can bind the correct keys in the osu! settings.
```

Comment:
```suggestion
The [Wii](https://en.wikipedia.org/wiki/Wii "Wikipedia") Dance Mat requires a [GameCube](https://en.wikipedia.org/wiki/GameCube "Wikipedia") controller adaptor to function on a computer. Once you have connected the dance pad to your computer, you can bind the correct keys in the osu! settings.
```
**An image of a GameCube adadptor for the Wii should prob be included.** This change is a bit of nitpick. Ignore it if you disagree.